todo-app/src/stores/TodosStore.js

```diff
@@ -0,0 +1,70 @@
+import todos from "../data/todos.json";
+import { observable, action, configure } from "mobx";
+
+configure({ enforceActions: true });
+
+
+const TodoItems = observable({
```

Comment:
While this does "work", it's not really the correct way to be working with MobX without decorators. Check out [this section of the docs](https://mobx.js.org/best/decorators.html).

Instead of just wrapping your `todos` array in `observable`, you're wrapping the entire object. Moreover, it's customary to use classes, not plain objects.